src/modules/governance/ProposalListItem.tsx

```diff
@@ -64,6 +66,17 @@ export function ProposalListItem({
             checked={diffReached}
             loading={mightBeStale}
           />
+          {minQuorumVotes === 1040000 ? ( // Long executor
```

Comment:
Is there no data within the AIP metadata to track if it's a long executor? I found this but not sure if it's used: https://github.com/aave/interface/blob/bea86a4081bd9124b3414cf90f30d7d95d1a435c/src/ui-config/governanceConfig.ts#L42

Reply:
Yeah that would be a better way to check, since the quorum is more likely to be updated than the executor

@foodaka It should be possible to check `targets` for this address, similar to the cross-chain PR

Reply:
yeah its on the exectuor on the proposal obect, took some time to find it 😓 
fixed in `4ed1c295e03d232e722057ee0a4c04dcd582ead5`